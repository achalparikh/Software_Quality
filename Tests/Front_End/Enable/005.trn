10_John_Doe_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _