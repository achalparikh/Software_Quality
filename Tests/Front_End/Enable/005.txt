login
standard
John Doe 
enable
John Doe
00001
logout