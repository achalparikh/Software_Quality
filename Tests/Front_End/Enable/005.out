Standard users cannot use the 'enable' transaction.